### PR TITLE
Added GI Packaging for Greenplum

### PR DESCRIPTION
**Pull request by Ashi-Jain1**
Signed-off-by: Ashi Jain <Ashi.Jain1@ibm.com>

### CODE REVIEW

File: verifiedUCPlugins.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Filter plugins read to zip:
 filter-plugin/logstash-filter-dynamodb-guardium/DynamodbOverCloudwatchPackage
 filter-plugin/logstash-filter-mysql-aws-guardium/MysqlOverCloudwatchLogsPackage
+filter-plugin/logstash-filter-onPremGreenplumdb-guardium/GreenplumdbOverFilebeatPackage
```
Comment (RefaelAdi) [inline verifiedUCPlugins.txt:4]: This change is required for SaaS adjustments only, but a few more things are needed to adjust this plug-in for SaaS env.
Let's remove this change and adjust the plug-in separately.

**Review by RefaelAdi — CHANGES_REQUESTED** [review]

**Review by RefaelAdi — APPROVED** [review]

**Review by AlexaNahum — COMMENTED** [review]
no content corrections

File: filter-plugin/logstash-filter-onPremGreenplumdb-guardium/GreenplumdbOverFilebeatPackage/gi_templates.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "plugin_name": "GreenPlumDB Over Filebeat",
```
Comment (Tal-Daniel) [inline filter-plugin/logstash-filter-onPremGreenplumdb-guardium/GreenplumdbOverFilebeatPackage/gi_templates.json:2]: I suggest to fix this to Greenplum, as in https://greenplum.org, especially if this is text is shown to users in the UI.

**Review by Tal-Daniel — CHANGES_REQUESTED** [review]

File: filter-plugin/logstash-filter-onPremGreenplumdb-guardium/GreenplumdbOverFilebeatPackage/gi_templates.json (hunk 2 above)
Comment (Ashi-Jain1) [reply filter-plugin/logstash-filter-onPremGreenplumdb-guardium/GreenplumdbOverFilebeatPackage/gi_templates.json:2]: Hello @Tal-Daniel , this text is not shown to users, User will see "GreenPlumDB" as a plugin name.
PFA screenshot .
Thanks

image

Comment (Tal-Daniel) [reply filter-plugin/logstash-filter-onPremGreenplumdb-guardium/GreenplumdbOverFilebeatPackage/gi_templates.json:2]: Thanks for checking. I would change all user facing mentions to "Greenplum".

Comment (Ashi-Jain1) [reply filter-plugin/logstash-filter-onPremGreenplumdb-guardium/GreenplumdbOverFilebeatPackage/gi_templates.json:2]: The changes have been done; plug-in name changed to "Greenplum".
PFA screenshot.
image
Thanks

**Review by Tal-Daniel — APPROVED** [review]

**Review by RefaelAdi — APPROVED** [review]

**Pull request merged**